### PR TITLE
Release Fixes

### PR DESCRIPTION
**Pull request by FBartos**
This should fix the following issues. The problem is that when I build on Linux and use the development module, the rstanarm package is causing problems and I cannot test the changes (also, I could not reproduce the covergence issue -  there might be different settings on macOS and linux? but the error message should be caught now).

might fix: https://github.com/jasp-stats/INTERNAL-jasp/issues/2225
might fix: https://github.com/jasp-stats/INTERNAL-jasp/issues/2226

### CODE REVIEW

File: R/MixedModelsCommon.R [hunk 1]
```diff
@@ -2682,23 +2682,22 @@ gettextf <- function(fmt, ..., domain = NULL)  {
...
 
-  thm  <- rstan:::rstanvis_theme()
+  #thm  <- rstan:::rstanvis_theme()
   clrs <- rep_len(rstan:::rstanvis_aes_ops("chain_colors"), plotData$nchains)
   base <- ggplot2::ggplot(plotData$samp,ggplot2::aes_string(x = "mcmcSamples", y = "value", color = "chain"))
 
   graph <- base + ggplot2::geom_path() + ggplot2::scale_color_manual(values = clrs) +
-    ggplot2::labs(x = "", y = levels(plotData$samp$parameter)) + thm
-
+    ggplot2::labs(x = "", y = levels(plotData$samp$parameter))
```
Comment (vandenman) [inline R/MixedModelsCommon.R:2700]:
```suggestion
    ggplot2::labs(x = NULL, y = levels(plotData$samp$parameter))
```

**Comment by vandenman** [comment]
Please ping me after the release so we can continue this then

**Comment by vandenman** [comment]
local changes for later:
```diff
-  stan_glmer  <- rstanarm::stan_glmer
+  stan_glmer  <<- rstanarm:::stan_glmer # FIXME!
+  stan_lmer   <<- rstanarm:::stan_lmer # FIXME!
 
   if (!is.null(jaspResults[["mmModel"]]))
     return()
@@ -1987,6 +1988,20 @@ gettextf <- function(fmt, ..., domain = NULL)  {
 
   modelFormula <- .mmModelFormula(options, dataset)
 
+  # here be dragons
+  original <- stanova::stanova
+  expr0 <- as.expression(body(stanova::stanova))
+  # replace "rstanarm::stan_" with "rstanarm:::stan_"
+  if (identical(expr0[[1]][[5]][[3]][[2]][[2]], "rstanarm::stan_")) {
+    expr0[[1]][[5]][[3]][[2]][[2]] <- "rstanarm:::stan_"
+  } else {
+    warning("stanova changed and the hack above no longer works! Expect crashes!", domain = NA)
+  }
+  replacement <- stanova::stanova
+  body(replacement) <- expr0
+
+  jaspBase:::assignFunctionInPackage(replacement, "stanova", "stanova")
+
   if (type == "BLMM") {
 
     model <- try(stanova::stanova(
@@ -2526,14 +2541,18 @@ gettextf <- function(fmt, ..., domain = NULL)  {
     )
 
     if (options$mcmcDiagnosticsType %in% c("histogram", "density")) {
-      p <- jaspGraphs::themeJasp(p, sides = "b")
+      p <- p +
+        jaspGraphs::geom_rangeframe(sides = "b") +
+        jaspGraphs::themeJaspRaw()
       p <- p + ggplot2::theme(
         axis.title.y = ggplot2::element_blank(),
         axis.text.y  = ggplot2::element_blank(),
         axis.ticks.y = ggplot2::element_blank()
       )
     } else {
-      p <- jaspGraphs::themeJasp(p)
+      p <- p +
+        jaspGraphs::geom_rangeframe() +
+        jaspGraphs::themeJaspRaw()
     }

```

**Comment by FBartos** [comment]
@vandenman 

Maybe we should merge this in? I completely forgot that this existed

**Comment by vandenman** [comment]
ehh sure, let me rerun the tests once to make sure.

**Comment by vandenman** [comment]
tests miraculously pass now! :confetti_ball:

**Review by vandenman — APPROVED** [review]

**Pull request merged**